src/MusicalScore/Player/MusicSheetPlayer.ts

```diff
@@ -0,0 +1,63 @@
+import {MusicSheet} from "../MusicSheet";
+import {Cursor} from "../../OpenSheetMusicDisplay";
+
+export class MusicSheetPlayer {
+    get IsPlay(): boolean {
+        return this.isPlay;
+    }
+    constructor(bpm: number, musicSheet: MusicSheet, cursor: Cursor, playCallback: (arr: Array<any>) => void) {
```

Comment:
Why the anonmyos callback? There should be at least an interface and a description what this does